public/checkPassword.js

```diff
@@ -0,0 +1,20 @@
+const form = document.forms["register"];
+const password = form["password"];
+const warningMessage = document.getElementById("register-warning-message");
+const passwordLessThanSixCharacters = document.createElement("p");
+warningMessage.appendChild(passwordLessThanSixCharacters);
+
+form.addEventListener("input", (event) => {
+  if (event.target.value.length < 6) {
+    passwordLessThanSixCharacters.textContent =
+      "Your password is less than 6 characters long!";
+  } else {
+    passwordLessThanSixCharacters.textContent = "Password length: ✅";
+  }
+});
```

Comment:
My understanding of the `input` event is that it's triggered on `<input>` elements rather than `<form>` elements.

If this does work, however, I have a suspicion that it may check the username is 6 characters long as well?

Reply:
Ah that's right, I'll have a look at fixing this 👍